src/analyze/analyze.c

```diff
@@ -457,15 +457,13 @@ static int acquire_host_info(sd_bus *bus, struct host_info **hi) {
                                    "/org/freedesktop/hostname1",
                                    hostname_map,
                                    host);
-        if (r < 0)
-                goto fail;
 
-        r = bus_map_all_properties(bus,
+        s = bus_map_all_properties(bus,
```

Comment:
Maybe that's a nit, but I really don't like variables called `s` to contain integers.